src/ray/gcs/store_client/redis_store_client.cc

```diff
@@ -503,7 +510,7 @@ bool RedisDelKeyPrefixSync(const std::string &host,
   });
 
   auto status = cli->Connect(io_service);
-  RAY_CHECK_OK(status) << "Failed to connect to redis";
+  RAY_CHECK(status.ok()) << "Failed to connect to redis: " << status.ToString();
```

Comment:
why not use RAY_CHECK_OK?

Reply:
Before https://github.com/ray-project/ray/pull/49391 the RAY_CHECK_OK will not print the message of the status. Now it does. reverted this line.